stream-chat-android-ui-components/src/main/res/layout/stream_ui_item_message_plain_text.xml

```diff
@@ -62,6 +64,17 @@
         app:layout_constraintTop_toBottomOf="@+id/messageText"
         />
 
+    <ImageView
+        android:id="@+id/deliveryFailedIcon"
+        android:layout_width="24dp"
+        android:layout_height="24dp"
+        android:src="@drawable/stream_ui_ic_warning"
+        android:visibility="gone"
```

Comment:
I just noticed that in other cases visibility is set to `visible` (it probably doesn't matter because we still configure that in decorator)

Reply:
Updated it so that it's `gone` by default everywhere 👍